### ISSUE TITLE
[Feature Request] Cloud saves

### ISSUE THREAD

**Issue by LemuelDayrit** [Ryujinx/Ryujinx#7195]
### Overview

i dont know if its possible to add implement cloud saves using github account and put it on the repo

### Smaller details

None

### Nature of request

is possible to add cloud saves in every game 

### Why would this feature be useful?

this feature should be useful when switching devices so it doesn't take too long to transfer saves manually

**Comment by gdkchan** [comment]
> i dont know if its possible to add implement cloud saves using github account and put it on the repo

Are you suggesting that we upload the saves on a github repository using the user's account? Github is not made for that, so I don't think it is a good idea. We'd probably need our own service/storage to manage the saves.

**Comment by Rafacasari** [comment]
Can't you just put the /saves/ folder to be synced with OneDrive?
If I'm not mistaken, you can even put the entire Ryujinx in OneDrive (so you don't have to always reconfigure it) using the [Ryujinx Portable Mode](https://github.com/Ryujinx/Ryujinx/wiki/Ryujinx-Setup-&-Configuration-Guide#portable-mode)

**Comment by PoloNX** [comment]
Yes or you can use rclone and cron to sync it every days for example

> Can't you just put the /saves/ folder to be synced with OneDrive? If I'm not mistaken you can even put the entire Ryujinx in OneDrive (so you don't have to always reconfigure it) using the [Ryujinx Portable Mode](https://github.com/Ryujinx/Ryujinx/wiki/Ryujinx-Setup-&-Configuration-Guide#portable-mode)

**Comment by Blue-dev93** [comment]
Best way to get the same result is using Syncthing. Requires a bit of an setup but when done it automatically syncs between the devices. The only downside is that you need a host device to sync properly.

**Comment by Edgecrusherr160** [comment]
Offering a refined UI experience for Portable Mode would be nice. Also, building in a simple way to choose OneDrive, iCloud, Dropbox, etc would be great. Sounds more like a longer term goal, though. Still, I can see the value in this.
